### ISSUE TITLE
Action SendToDesktop behaves differently from Openbox

### ISSUE THREAD

**Issue by jech** [labwc/labwc#841]
In Openbox, action `SendToDesktop` switches to the destination desktop.  In labwc, it does not.  So where in Openbox I could say:
```
    <keybind key="S-A-Left">
      <action name="SendToDesktop">
        <to>left</to>
        <wrap>no</wrap>
      </action>
    </keybind>
```
in labwc, I must say:
```
    <keybind key="S-A-Left">
      <action name="SendToDesktop">
        <to>left</to>
        <wrap>no</wrap>
      </action>
      <action name="GoToDesktop">
        <to>left</to>
        <wrap>no</wrap>
      </action>
    </keybind>
```
That's fine, but I'm wondering if the behaviour is deliberate.

(Additionally, `<wrap>no</wrap>` doesn't seem to do anything useful, but that's a different issue.)

**Comment by johanmalm** [comment]
Good spot. That’s not deliberate (from me anyway 😄 )

Openbox spec has a “follow” element which defaults to yes. I vote for copying that approach, even if it is a small “breakage”.

http://openbox.org/wiki/Help:Actions#SendToDesktop

**Comment by johanmalm** [comment]
Fixed by #851
